mapit_gb/management/commands/mapit_UK_add_slugs_to_local_authorities.py

```diff
@@ -0,0 +1,34 @@
+# This script is for a one off import of govuk slugs for local authorities
+
+import json
+import os.path
+
+from django.core.management.base import NoArgsCommand
+from mapit.models import Area, CodeType
+
+
+class Command(NoArgsCommand):
+    help = 'Assigns slugs from frontend to local authorities'
+
+    def handle_noargs(self, **options):
+        code_type, _ = CodeType.objects.get_or_create(
+            code='govuk_slug',
+            defaults={'description': 'Slug for use by GOV.UK'}
+        )
+
+        file_path = os.path.dirname(__file__) + '/../../data/authorities.json'
+        authorities = json.load(open(file_path))
```

Comment:
It's good practice to [use `with` when opening files](http://preshing.com/20110920/the-python-with-statement-by-example/), because it ensures that the file is closed again if any exception occurs:

```python
with open(file_path) as f:
    authorities = json.load(f)
```